Assets/Scripts/UIScripts/PauseScreen.cs

```diff
@@ -8,6 +10,7 @@ public class PauseScreen : MonoBehaviour
     public MusicPlayer musicPlayer;
     public GameObject playerMovement;
     private PlayerMovement _playerMovementScript;
+    public CountdownManager countdownManager;
```

Comment:
I think we can avoid adding this attribute CountdownManager by just making CountdownManager a singleton (check the other manager scripts) and referencing it as CountdownManager.Current.SetCountdown()